overlord/snapstate/handlers.go

```diff
@@ -642,6 +643,28 @@ func hasOtherInstances(st *state.State, instanceName string) (bool, error) {
 	return false, nil
 }
 
+var ErrKernelGadgetUpdateTaskMissing = errors.New("cannot complete refreshing kernel through an old snapd change without a gadget update task")
```

Comment:
```suggestion
var ErrKernelGadgetUpdateTaskMissing = errors.New("cannot refresh kernel with change created by old snapd that is missing gadget update task")
```

how does that read / sound ?